release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -68,6 +68,17 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-12-networking"]
 === Networking
 
+[id="ocp-4-12-openshift-sdn-ovn-kubernetes-feature-migration-support"]
+==== OpenShift SDN to OVN-Kubernetes feature migration support
+
+If you migrate from the OpenShift SDN cluster network provider to the OVN-Kubernetes cluster network provider, you can migrate your configurations for the following capabilities:
+
+* Egress IP addresses configured with the automatically assigned approach
+* Egress network policies
```

Comment:
I think it would be better to use 'Egress Firewall' here since it's the feature's name in both OpenShift SDN and OVN Kubernetes.